apps/google-analytics-4/frontend/src/components/common/HyperLink/HyperLink.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { MouseEventHandler } from 'react';
+import { TextLink } from '@contentful/f36-components';
+
+interface Props {
+  body: string;
+  substring: string;
+  onClick?: MouseEventHandler<HTMLAnchorElement>;
+  hyperLinkHref?: string;
```

Comment:
nit: What is the benefit of making the link optional?

Reply:
my intention here is that sometimes we will want a component where an href link is provided and sometimes we have an onClick event to pass. For example, the hyperlink currently used to open the app config page passes an onClick JS function to open the config page. In other cases, a link such as Google Analytics will be opened. Thoughts?